src/components/SendButton/sendButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './sendButton.module.css';
+import { IconContext } from 'react-icons';
+import { FiArrowRight } from 'react-icons/fi';
+
+interface Props {
+  status: string;
```

Comment:
This links to the above comment, this can be typed more strongly in a typescript-y way.